app/src/main/java/com/lightricks/feedexercise/ui/feed/FeedViewModel.kt

```diff
@@ -36,7 +36,7 @@ open class FeedViewModel : ViewModel() {
  * dependencies into [FeedViewModel] in the future, then this is the place to do it.
  */
 class FeedViewModelFactory : ViewModelProvider.Factory {
```

Comment:
I think we should also change to it to:
`class FeedViewModelFactory(private val context: Context) : ViewModelProvider.NewInstanceFactory() {`